content/en/observability_pipelines/guide/run_multiple_pipelines_on_the_same_host.md

```diff
@@ -0,0 +1,126 @@
+
+## Prerequisites
+
+[Set up the first pipeline](https://docs.datadoghq.com/observability_pipelines/set_up_pipelines/?tab=pipelineui) and install the Worker on your host.
+
+## Create another pipeline
+
+[Set up another pipeline](https://docs.datadoghq.com/observability_pipelines/set_up_pipelines/?tab=pipelineui) for the additional Worker that you want to run on the same host. When you get to the Install page, follow the below steps to run the additional Worker.
+
+## Run an additional Worker
+
+When you installed the first Worker, by default you have:
+
+- The service binary: `/usr/bin/observability-pipelines-worker`
+- The service definition file: `/lib/systemd/system/observability-pipelines-worker.service`, which looks like:
+
+    ```
+    [Unit]
+    Description="Observability Pipelines Worker"
+    Documentation=https://docs.datadoghq.com/observability_pipelines/
+    After=network-online.target
+    Wants=network-online.target
+
+    [Service]
+    User=observability-pipelines-worker
+    Group=observability-pipelines-worker
+    ExecStart=/usr/bin/observability-pipelines-worker run
+    Restart=always
+    AmbientCapabilities=CAP\_NET\_BIND\_SERVICE
+    EnvironmentFile=-/etc/default/observability-pipelines-worker
+
+    [Install]
+    WantedBy=multi-user.target
+    ```
+- An environment file: `/etc/default/observability-pipelines-worker`, which looks like:
+
+    ```
+    DD\_API\_KEY=\<datadog\_api\_key\>
+    DD\_SITE=datadoghq.com
+    DD\_OP\_DESTINATION\_DATADOG\_ARCHIVES\_AWS\_ACCESS\_KEY\_ID=\<redacted\>
+    DD\_OP\_DESTINATION\_DATADOG\_ARCHIVES\_AWS\_SECRET\_ACCESS\_KEY=\<redacted\>
+    DD\_OP\_PIPELINE\_ID=\<pipeline\_id\>
+    DD\_OP\_SOURCE\_DATADOG\_AGENT\_ADDRESS=0.0.0.0:8282
+    DD\_OP\_API\_ENABLED=true
+    ````
+
+- A data directory: `/var/lib/observability-pipelines-worker`
+
+### Configure an additional Worker
+
+In this example, another pipeline was created with the Fluent source. To configure a Worker for the pipeline:
+
+1. Run the following command to create a new data directory, updating `op-fluent` to a directory name that fits your use case:
+    ```
+    sudo mkdir /var/lib/op-fluent
+    ```
+1. Run the following command to change the owner of the data directory to `observability-pipelines-worker:observability-pipelines-worker`. Make sure to update `op-fluent` to your specific directory name.
+    ```
+    sudo chown -R observability-pipelines-worker:observability-pipelines-worker /var/lib/op-fluent/
+    ```
+1. Create an environment file for the new systemd service, such as `/etc/default/op-fluent` where `op-fluent` is replaced with your specific filename. An example of the file content:
+    ```
+    DD_API_KEY=<datadog_api_key>
+    DD_OP_PIPELINE_ID=<pipelines_id>
+    DD_SITE=[datadoghq.com](http://datadoghq.com)
+    DD_OP_DESTINATION_DATADOG_ARCHIVES_AWS_ACCESS_KEY_ID=<aws_access_key_id>
```

Comment:
We talked about removing these I think, did you want to keep them here? I think we did away with these via IAM roles or similar?

Reply:
Hey @ckelner! Sorry, I hadn't made the updates yet when reviewed this. I should have now removed all env vars that we said to remove.